pkg/wshutil/wshproxy.go

```
@@ -127,28 +181,30 @@
-		authRtn, err := router.HandleProxyAuth(origMsg.Data)
-		if err != nil {
-			respErr := fmt.Errorf("error handling proxy auth: %w", err)
-			p.sendResponseError(origMsg, respErr)
-			return "", respErr
-		}
-		p.SetAuthToken(authRtn.AuthToken)
-		announceMsg := RpcMessage{
-			Command:   wshrpc.Command_RouteAnnounce,
-			Source:    authRtn.RouteId,
-			AuthToken: authRtn.AuthToken,
+		if origMsg.Command == wshrpc.Command_AuthenticateToken {
```

golangci-lint (1.62.2)

SA9003: empty branch

(staticcheck)